src/server/json_family.cc

```diff
@@ -106,6 +107,56 @@ ParseResult<WrappedJsonPath> ParseJsonPath(std::string_view path) {
 
 }  // namespace json_parser
 
+namespace reply_generic {
+
+template <typename T> void Send(RedisReplyBuilder& rb, T value) = delete;
+
+template <> void Send(RedisReplyBuilder& rb, std::size_t value) {
```

Comment:
our coding standards dictate that mutable objects are passed by pointer and they should be the last ones